src/lsstseries/ensemble.py

```diff
@@ -284,6 +284,66 @@ def prune(self, threshold=50, col_name=None):
 
         return self
 
+    def bin_sources(self, time_window=1.0, additional_cols=None, use_map=True, **kwargs):
+        """Bin sources on within a given time range to improve the estimates.
+
+        Notes
```

Comment:
In general the LSST code that I have seen has Notes at the bottom, after Parameters. Having said that, I think it is only a custom and not obligatory so I am fine if you leave it like this as well.

Reply:
I'd prefer to stay consistent with LSST code. I moved the notes section.